custom_components/tuya_local/devices/luko_dehumidifier.yaml

```diff
@@ -0,0 +1,65 @@
+name: Luko Dehumidifier
+products:
+  - id: 4w2fzgr62c1oyjsx
```

Comment:
Product should have a name, containing branding (including model name or number if available)

Top level name above should be generic, without branding, in case of other devices matching the same config.